ts/BUILD.bazel

```diff
@@ -84,49 +84,47 @@ config_setting(
 config_setting(
     name = "verbose_flag",
     flag_values = {
-        ":verbose": "true"
-    }
+        ":verbose": "true",
+    },
 )
 
 config_setting(
     name = "supports_workers_flag",
     flag_values = {
-        ":supports_workers": "true"
-    }
+        ":supports_workers": "true",
+    },
 )
 
 options(
     name = "options",
-    visibility = ["//visibility:public"],
```

Comment:
Can we avoid all the formatting changes? For a one line change this file has quite the diff, especially in a non-main branch where this needs to be merged :/

Reply:
The way to avoid it would be to force the previous committer on this file to have formatted it. We don't use Aspect Workflows and haven't bothered with a buildifier test on our "simple" OSS repos.

Reply:
But we haven't done that :(

It just makes the diff ugly for such a tiny actual change, and when in a feature branch increases the likelihood of merge conflicts. I'd prefer one big format commit in `main` but 🤷

Reply:
Sure we can just format main. Maybe it's time to add buildifier assertion in rules-template and everything we cloned from it